### PR TITLE
 [T2] Resource graph

### PR DESCRIPTION
**Pull request by zhoxing-ms**

### CODE REVIEW

File: sdk/resources/azure-mgmt-resourcegraph/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 2.2.0 (2020-10-16)
```
Comment (msyyc) [inline sdk/resources/azure-mgmt-resourcegraph/CHANGELOG.md:3]: According the result of last meeting, version number of python sdk for track should begin with "track1 version + 5". So the version number should be 7.0.0. (remember change _version.py as well)

File: tools/azure-sdk-tools/sdk_packaging.toml [hunk 2]
```diff
@@ -1,2 +1,4 @@
 [packaging]
-auto_update = false
```
Comment (jsntcy) [inline tools/azure-sdk-tools/sdk_packaging.toml:2]: Why do we need update this common file?

**Review by jsntcy — APPROVED** [review]

**Pull request merged**